Dockerfile

```diff
@@ -13,14 +13,8 @@ COPY requirements.txt /app/
 # Install the dependencies
 RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
We need to change this to use poetry

Reply:
Take a look at [how I did Dockerfile with poetry](https://github.com/zainaraza43/the-house/blob/main/Dockerfile)